vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util/helpers.go

```diff
@@ -154,7 +155,13 @@ func checkErr(err error, handleErr func(string)) {
 
 func statusCausesToAggrError(scs []unversioned.StatusCause) utilerrors.Aggregate {
 	errs := make([]error, len(scs))
+	errorMsgs := sets.NewString()
 	for i, sc := range scs {
+		// check for duplicate error messages and skip them
+		if errorMsgs.Has(sc.Message) {
+			continue
```

Comment:
if we ever skip because of this, that means `errs` has nil errors at the end, which I would expect to panic when NewAggregate attempts to call Error on them